CRM/Contact/BAO/Group.php

```diff
@@ -350,6 +350,23 @@ public static function create(&$params) {
       CRM_Utils_Hook::pre('create', 'Group', NULL, $params);
     }
 
+    //  CRM-287 Disable child groups if all parents are disabled.
```

Comment:
CRM-287? This is not the correct issue number!

Reply:
I think this whole process should be done after parent group is disabled.

Reply:
Updated the issue number.

Reply:
And about the workflow. Lets consider a situation with three parent groups to one child group. The child group will be disabled after the 3rd parent is disabled. But when any of the parent is enabled the child should be enabled too. So I think the current workflow is right.

Reply:
Can you iron out more of my confusion for below use case(or isn't a valid use case)?
1. Disabled child group. 
2. After some days Disabled <code>#1's</code> Parent group.
3. Later Enabled <code>#1's</code> Parent group, but don't want the child group to be enabled.